test/integration/framework/master_utils.go

```diff
@@ -271,7 +271,8 @@ func NewIntegrationTestMasterConfigWithOptions(opts *MasterConfigOptions) *contr
 
 // MasterConfigOptions are the configurable options for a new integration test master config.
 type MasterConfigOptions struct {
-	EtcdOptions *options.EtcdOptions
+	EtcdOptions                    *options.EtcdOptions
+	ServeRemovedAPIsOneMoreRelease bool
```

Comment:
was this used?

Reply:
> was this used?

In one iteration it was attempted, but the wiring for the etcd storage test is actually via options, not via config.  The kube-apiserver wiring has never been fully untangled.  Though after removing the ssh tunnels it will get a little easier.